### PR TITLE
Merge Demo

### PR DESCRIPTION
**Pull request by Dellrodar**
* Added readme for merge demo

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Test file
```
Comment (Drethic) [inline readme.md]: This file could be written as markdown

Comment (Drethic) [inline readme.md]: Convention says readme should be all caps.

**Review by Drethic — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (Dellrodar) [reply readme.md]: Thanks, I'll change that. Fixed in newest commit.

Comment (Dellrodar) [reply readme.md]: changed in c04b479

**Review by Drethic — APPROVED** [review]

**Pull request merged**